core/src/main/java/org/apache/accumulo/core/singletons/SingletonManager.java

```diff
@@ -170,8 +170,10 @@ public static synchronized void setMode(Mode mode) {
       transitionedFromClientToConnector = true;
     }
 
-    // always allow transition to closed and only allow transition to client/connector when the
-    // current mode is not server
+    /**
```

Comment:
This isn't a javadoc comment.

```suggestion
    /*
```